### DISCUSSION TITLE
CameraCaptureUI - is it working in Windows App Sdk 1.1?

### DISCUSSION THREAD

**Discussion by nikolayvpavlov** [microsoft/microsoft-ui-xaml#7847] [Q&A]
I am trying to capture a picture using the camera of the computer. I follow the [example](https://learn.microsoft.com/en-us/windows/uwp/audio-video-camera/capture-photos-and-video-with-cameracaptureui).  

The thing is, that `await captureUI.CaptureFileAsync(CameraCaptureUIMode.Photo)` always returns null, and the UI window for the camera capture is not displayed.

This happens when the app is run as packaged and as unpackaged.  I tried it with Windows App Sdk 1.0 and 1.1.5.  

Does anyone know if `captureUI.CaptureFileAsync()` is actually supported by Windows App Sdk?

**Comment by castorix** [comment]
There are other threads about this problem, like [#1034](https://github.com/microsoft/WindowsAppSDK/issues/1034)
I use DirectShow for camera capture (or Microsoft Media Foundation)
